### PR TITLE
try testing cli

### PR DESCRIPTION
**Pull request by kain88-de**
@mimischi this doesn't seem to work correctly. The folder I'm trying to execute the command in isn't generated or I'm not switching into it for some reason. This means I can't check if all the files I want are generated.

Changes made in this Pull Request:
 - 

PR Checklist
------------
 - ~[ ] CHANGELOG updated (added [news fragment](https://github.com/hawkowl/towncrier#news-fragments) into changelog directory)?~
 - ~[ ] Issue raised/referenced?~

### CODE REVIEW

**Comment by kain88-de** [comment]
it works at least locally now

**Comment by kain88-de** [comment]
@mimischi can you review this.

File: mdbenchmark/tests/test_generate.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import os
+from glob import glob
+
+import click
+import numpy as np
+from numpy.testing import assert_array_equal
+
+import pytest
+import os
```
Comment (mimischi) [inline mdbenchmark/tests/test_generate.py]: You're doing `import os` twice

**Comment by kain88-de** [comment]
@mimischi can you review the latest changes

**Comment by mimischi** [comment]
Looks good to me. I'm OK merging.

**Pull request merged**